_data/projects/JavaScript.yml

```diff
@@ -0,0 +1,12 @@
+name: JavaScript
+desc: Repository containing various JavaScript cose
```

Comment:
@kshitizsaini113 can we be a bit more descriptive here about what the project is for? It's not clear to me from the repository itself how contributors can help out.

Reply:
@shiftkey I have updated the description. You may have a look at it.